astropy/convolution/kernels.py

```diff
@@ -145,11 +145,13 @@ class Gaussian2DKernel(Kernel2D):
     """
     _separable = True
     _is_bool = False
-
-    def __init__(self, stddev, **kwargs):
-        self._model = models.Gaussian2D(1. / (2 * np.pi * stddev ** 2), 0,
-                                        0, stddev, stddev)
-        self._default_size = _round_up_to_odd_integer(8 * stddev)
+   
+    def __init__(self, width, height=width, theta=0.0, support_scaling=8, **kwargs):
```

Comment:
You can't use width yet at this point, I think the best is to say ``height=None``, and then in the next line test for it: if ``None`` set it to ``width``.